apps/www/src/lib/registry/default/ui/command/CommandGroup.vue

```diff
@@ -11,7 +11,7 @@ const props = defineProps<ComboboxGroupProps & {
 <template>
   <ComboboxGroup
     v-bind="props"
-    :class="cn('overflow-hidden p-1 text-foreground', $attrs.class ?? '')"
```

Comment:
thanks for the PR @ahmedmayara . Howewer can we try not to edit styling in registry? We wanna keep the same styling as ui.shadcn's components.

Reply:
I removed the overflow-hidden class because the command palette doesn't work properly, the scrollbar doesn't show, which means the users can't navigate through the command component.